contracts/LockedAccount.sol

```diff
@@ -144,6 +156,7 @@ contract LockedAccount is AccessControlled, AccessRoles, TimeSource, ReturnsErro
     // if used before longstop date, calculates penalty and distributes it as revenue
     function unlock()
         onlyStates(LockState.AcceptingUnlocks, LockState.ReleaseAll)
+        acceptAgreement(msg.sender)
```

Comment:
Each investor accepts agreement by calling lock(). We can safely remove it here

Reply:
I was errering on the side of safety because i wasn't sure. Removed it now.